### PR TITLE
Add draft minimum coding standards as website

### PR DESCRIPTION
**Pull request by RobinL**

### CODE REVIEW

**Comment by RobinL** [comment]
Hiya,

I've added a first draft of minimum coding standards, and an interactive webapp that allows the user to select the riskiness of their project, and get a set of minimum standards plus language specific guidance.

The guidance itself is in `web/data/minimum_standards_blurb.md` and `web/data/standards_table.csv`


I guess may be a good idea to merge this and then start to solicit comments, particularly on `standards_table.csv`

Robin

**Comment by EdwardAndress** [comment]
Sounds great! How does one use the webapp?

File: code_smells.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+### Too many arguments
+
+### Functions that contain too much code
+
+### Reliance on global scope
+
+
+### Rolling your own solution 
+
+### Long functions
```
Comment (EdwardAndress) [inline code_smells.md]: Is this the same as 'Functions that contain too much code?'

File: code_smells.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Functions should be short and have a single purpose. If you find yourself writing a function that’s over [around 50 lines of code](https://softwareengineering.stackexchange.com/questions/27798/what-should-be-the-maximum-length-of-a-function), then you should consider splitting it into a number of different functions. 
+
+### Magic numbers
+
+Don't have magic numbers - they are parameters.
```
Comment (EdwardAndress) [inline code_smells.md]: We don't define 'magic numbers' here

**Pull request merged**